src/app/instance.h

```diff
@@ -81,9 +81,7 @@ class Instance {
   void SetupTunnels(bool is_reloading = false);
 
   /// @brief Should remove old tunnels after tunnels config is updated
-  /// TODO(unassigned): not fully implemented
-  void RemoveOldTunnels(
-      std::vector<std::string>& updated_tunnels);
+  void RemoveOldTunnels(std::vector<std::string>& updated_tunnels);
```

Comment:
Sorry @anonimal , but i really don't see what's left to do. This function removes all tunnels that are not present in "updated_tunnels". What am i missing ?